contracts/button/lib.rs

```diff
@@ -161,8 +148,7 @@ pub trait ButtonGame {
     /// where k is the number of players that participated until the button has died
     /// Can be overriden to some other custom calculation
     fn pressiah_score(&self) -> Balance {
-        let presses = self.get().presses;
-        (presses * num::integer::sqrt(presses)) as Balance
+        (self.get().total_rewards) as Balance
```

Comment:
that is not what we established? We wanted to lower the Pressiah reward to subquadratic?

_as we decided to mint button tokens instead of distributing them from the predefined pool, we may actually use arbitrary function here, so how about using something subquadratic instead of the current proposal? I.e., sth like k*sqrt(k), which wouldn't be such a high percentage in case of a very long streak. (...) communicate it as a 'pressiah multiplier', i.e., the pressiah would get k due to clicking as k-th person in the row, and it will be multiplied by the multiplier, that could be a square root of k (rounded up)_

Reply:
I missed that part, I thought the pressiah is always supposed to get the half of total distributed rewards